### DISCUSSION TITLE
Room Assistant on Home Assistant

### DISCUSSION THREAD

**Discussion by carlbehrent** [mKeRix/room-assistant#1152] [Q&A]
I have two instances of Room Assistant in a cluster, one running on a Pi Zero W, and one running on Home Assistant itself.

The instance on the Pi Zero is running fine and updates as I go in and out of range.
The instance running on Home Assistant appears to be running fine as per the log file but normally displays the status of devices as 'not home'. If left for some time I notice it occassionaly updates and shows the correct status, but mostly it doesn't.

Any ideas?

**Comment by worm5406** [comment]
How close do you have to be to the HA to get it to see you?

**Comment by carlbehrent** [comment]
Somethings I can be within 1 metre of it and it doesnt show.
I think now it could be related to my iphone as my wifes Samsung phone seems to be more reliable and showing the correct status.

**Reply by worm5406** [reply]
I am trying to figure how to restart the device on unavailable. I think they are having communication problems and don't respond which is why we get issues.

uugh. Almost like a storm. Since I put them in some of my lights have been dropping off line too.
